### PR TITLE
Fix walkthrough

### PR DESCRIPTION
**Pull request by vaikas-google**
Previously I had not updated the walkthrough for
Service[Broker|Class|Plan] renames to ClusterService[Broker|Class|Plan]

This addresses that.

### CODE REVIEW

**Comment by pmorie** [comment]
Thanks

File: docs/walkthrough-1.7.md [hunk 1]
```diff
@@ -88,14 +88,14 @@ Notice that the `status` field has been set to reflect that the broker server's
...
 
-The controller created a `ServiceClass` for each service that the UPS broker
-provides. We can view the `ServiceClass` resources available in the cluster by
+The controller created a `ClusterServiceClass` for each service that the UPS broker
+provides. We can view the `ClusterServiceClass` resources available in the cluster by
 executing:
 
 ```console
-$ kubectl get serviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
+$ kubectl get clusterserviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
```
Comment (kibbles-n-bytes) [inline docs/walkthrough-1.7.md]: Oh my God hahah, I never noticed this line here before. So simple and elegant. 🤣

**Comment by kibbles-n-bytes** [comment]
Talked offline; @vaikas-google wants to do the rename in a follow-up.

**Comment by vaikas-google** [comment]
And just to expand on K&B comment above, I was hoping to have a quick fix to the walkthrough, which is currently broken. This would fix the walkthrough without requiring code changes and get our users back up and running. So, hence why was going to tackle the renaming of the files in a separate PR.

**Pull request merged**